app/models/views/application_overview.rb

```diff
@@ -2,8 +2,8 @@ module Views
   class ApplicationOverview
```

Comment:
This class is getting quite big now, we should probably consider splitting it as well as it seems to have more responsibilities already. Not now, of course, that's just a thought.